backend/expense/src/main/java/org/egov/digit/expense/web/models/IndividualSearchRequest.java

```diff
@@ -0,0 +1,36 @@
+package org.egov.digit.expense.web.models;
```

Comment:
@ajay-egov we can not copy IndividualSearchRequest model here. use Map to create these.